src/ui/components/fiber-tree/TreeLeafCaption.tsx

```diff
@@ -87,13 +90,24 @@ const TreeLeafCaptionContainer = React.memo(
       event.stopPropagation();
       pin(id);
     };
+    const handleMouseEnter = () => {
+      const channel = remoteSubscriber.ns("highlighter");
```

Comment:
Subscriber method calls should be moved into highlight context API

Reply:
Fixed